pkg/rtc/mediatracksubscriptions.go

```diff
@@ -197,16 +206,26 @@ func (t *MediaTrackSubscriptions) AddSubscriber(sub types.LocalParticipant, code
 
 	// when out track is bound, start loop to send reports
 	downTrack.OnBind(func() {
+		wr.DetermineReceiver(downTrack.Codec())
+		if err = wr.AddDownTrack(downTrack); err != nil {
+			logger.Errorw("could not add down track", err, "participant", sub.Identity(), "pID", sub.ID())
+		}
 		go subTrack.Bound()
 		go t.sendDownTrackBindingReports(sub)
+		// initialize to default layer
+		t.notifySubscriberMaxQuality(subscriberID, downTrack.Codec(), livekit.VideoQuality_HIGH)
```

Comment:
@boks1971 It has been moved to here now, since we can only get DownTrack's negotiated codec after it has been binded